<launch>

<!--rosbag#####################################################################################################-->
    <param name="/use_sim_time" value="true" />
    <node pkg="rosbag" type="play" name="rosbag" args="-l /home/luo/EX_ws/src/rosbag/lRGB_dynamic111020hz.bag" />

<!--pedol######################################################################################################-->


    <!-- Manager -->
    <node pkg="nodelet" type="nodelet" args="manager" name="Mg" output="screen"  >
        
    </node>

    

    <!-- VisualNodeletClass -->
    <node pkg="nodelet" type="nodelet" args="load pedol/VisualNodeletClass Mg" name="VisualNodeletClass" output="screen" >
        <param name="AgentId"        type="int"    value="0"/>
        <param name="AgentFrameId"   type="string" value="livox_frame" /> 
        <remap from="/VisualNodeletClass/in" to="/livox/lidar" />
    </node>

    <!-- PathDisplayNode -->
    <node pkg="nodelet" type="nodelet" args="load pedol/PathDispNodeleteClass Mg" name="PathDispNodeleteClass" output="screen" >
        <!-- <param name="/dataset_pub_delay"      type="double"  value="5.0" />
        <param name="/dataset_pub_rate"       type="int"     value="30" />
        <param name="/publish_gt"             type="bool"    value="true" /> -->
        <!-- <remap from="/PathDispNodeleteClass/in" to="/mavros/local_position/odom" /> -->
    </node>


    


<!-- rviz####################################################################################################### -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find pedol)/launch/rviz/pedol_visual.rviz" />

</launch>
